debug.go

```diff
@@ -107,11 +110,34 @@ const pageHTML = `
 			border-color: #3A3A3A;
 			background-color: #ffffff;
 		}
+		header::after {
```

Comment:
I think all this CSS creeping into Go files might be a sign it's time to use templating.